run_kraken.py

```diff
@@ -37,9 +39,39 @@ def main(cfg):
     logging.info("Starting kraken")
 
     # Parse and read the config
-    if os.path.isfile(cfg):
-        with open(cfg, "r") as f:
-            config = yaml.full_load(f)
+    # Initializing flag variable as 0 to test config is a file or not
+    flag = 0
+
+    try:
+        # Checking config is yaml file or not
+        check_yaml_file = urlparse(cfg)
+        path = check_yaml_file.path
+
+        if path[-5:] == ".yaml" or path[-4:] == ".yml":
+
+            # For config as local file
+            if os.path.isfile(cfg):
+                # Set flag as 1 since config is local file
+                flag = 1
+                with open(cfg, "r") as f:
+                    config = yaml.full_load(f)
+
+            # For config as remote file
+            else:
+                # Set flag as 1 since config is remote file
+                flag = 1
+                content = requests.get(cfg)
```

Comment:
Want to validate that the get request is getting a valid response, should check that the status_code is 200 here